Makefile

```diff
@@ -265,8 +265,12 @@ update-copyright:
 # dev container
 #############################
 
+# build devcontainer locally
+build-devcontainer:
+	devcontainer build --workspace-folder=.github --push=false --image-name="ghcr.io/bucketeer-io/bucketeer-devcontainer:latest"
```

Comment:
I think we can add the guideline for new contributor in readme 
that how to install `devcontainer`

Reply:
I'll add guideline to contribution docs 👍 
https://docs.bucketeer.io/contribution-guide/contributing